web/pdf_find_controller.js

```diff
@@ -736,6 +736,7 @@ class PDFFindController {
       state,
       previous,
       matchesCount: this._requestMatchesCount(),
+      query: this._query,
```

Comment:
Note that this won't actually return *exactly* what the findbar input contains, which may be surprising to the user.
What you actually want here is probably `rawQuery: this._rawQuery,` instead.